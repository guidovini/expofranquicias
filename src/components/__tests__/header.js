import React from 'react'
import { render } from '@testing-library/react'

import Header from '../header'

describe('Header', () => {
  test('Should_RenderHeaderCorrectly', () => {
    const { container } = render(<Header siteTitle="Default Starter" />)
    expect(container).toMatchSnapshot()
  })

  test('Should_RenderHeaderElements', () => {
    const { getByText, queryByText } = render(
      <Header siteTitle="Default Starter" />
    )
    expect(getByText('Expofranquicias Ecuador')).toBeInTheDocument()
    expect(getByText('Franquicias Disponibles')).toBeInTheDocument()
    expect(queryByText('Buscar Franquicias')).not.toBeInTheDocument()
    expect(queryByText('Direcci√≥n')).not.toBeInTheDocument()
  })
})
