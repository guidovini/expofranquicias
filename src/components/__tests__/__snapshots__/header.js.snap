// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  aria-label="main navigation"
  className="navbar is-link"
  role="navigation"
>
  <div
    className="navbar-brand"
  >
    <a
      className="navbar-item"
      href="/"
    >
      <header
        className="title is-4 has-text-light has-text-weight-light"
      >
        Expofranquicias Ecuador
      </header>
    </a>
    <div
      className="navbar-burger burger"
      onClick={[Function]}
    >
      <span />
      <span />
      <span />
    </div>
  </div>
  <div
    className="navbar-menu"
  >
    <div
      className="navbar-start"
    />
    <div
      className="navbar-end"
    >
      <a
        className="navbar-item is-size-6"
        href="/"
      >
        <p>
          Inicio
        </p>
      </a>
      <a
        className="navbar-item is-size-6"
        href="/franquicias-disponibles"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-briefcase"
          />
        </span>
        <span>
          Â 
        </span>
        <span>
          Franquicias Disponibles
        </span>
      </a>
      <a
        className="navbar-item is-size-6"
        href="/contacto"
      >
        Contacto
      </a>
    </div>
  </div>
</nav>
`;
